src/pat/modal/modal.js

```diff
@@ -119,22 +119,15 @@ define([
         },
 
         resize: function() {
-            var modal_height = this.$el.outerHeight(true);
-            var modal_padding = modal_height - this.$el.outerHeight();
-            var max_height = $(window).innerHeight() - modal_padding;
-            var $tallest_child = this.getTallestChild();
-            var tallest_child_height = $tallest_child.outerHeight(true);
-            var header_size = this.$el.find('.header').outerHeight(true);
+            
+            var panel_content_elem = this.$el.find(".panel-content");
+            var header_elem = this.$el.find('.header');
 
-            if (tallest_child_height !== modal_height) {
-                modal_height = tallest_child_height + header_size + modal_padding;
-            }
-            if (max_height < modal_height) {
+            var modal_height = panel_content_elem.outerHeight(true) + header_elem.outerHeight(true);
+
+            if (this.$el.height() < modal_height) {
                 this.$el.addClass("max-height").css("height", max_height);
```

Comment:
@pilz we have an error here that prevents the display of the modal.

```
patterns.pat.modal: Failed while initializing 'modal' pattern. ReferenceError: max_height is not defined
```

The fix is either to set `max_height` to a proper value or '100%' . I am making a PR for this.